app/backend/src/couchers/urls.py

```diff
@@ -36,7 +36,7 @@ def account_settings_link():
 
 
 def password_reset_link(*, password_reset_token):
-    return f"{config['BASE_URL']}/complete-password-reset?token={password_reset_token}"
+    return f"{config['BASE_URL']}/set-new-password?token={password_reset_token}"
```

Comment:
before we reset the password and then put a new one later on... now we put a new one directly, that's why I think the name change gives it more clarity, besides the change is different from how it was before.

open to comments

Reply:
"Password reset" is the usual term used in this context, so we should stick to that.

Reply:
the thing is, for the sake of clarity I would set a new name, because there is already a password reset page which is only for putting the email, (image attached)

a different name could help to differentiate 

![image](https://github.com/Couchers-org/couchers/assets/16777161/2684661d-3ec0-45f4-a53a-b1a025d32d33)

Reply:
also we avoid using old references to the previous code

Reply:
still if isn't enough I can revert it to the previous name